src/api/telemetry/TelemetryCollection.js

```diff
@@ -24,15 +24,27 @@ import _ from 'lodash';
 import EventEmitter from 'EventEmitter';
 import { LOADED_ERROR, TIMESYSTEM_KEY_NOTIFICATION, TIMESYSTEM_KEY_WARNING } from './constants';
 
+/**
+ * @typedef {import('../objects/ObjectAPI').DomainObject} DomainObject
+ */
+
+/**
+ * @typedef {import('../time/TimeContext').TimeContext} TimeContext
+ */
+
+/**
+ * @typedef {import('../../../openmct').OpenMCT} OpenMCT
+ */
+
 /** Class representing a Telemetry Collection. */
 
 export default class TelemetryCollection extends EventEmitter {
     /**
      * Creates a Telemetry Collection
      *
      * @param  {OpenMCT} openmct - Open MCT
-     * @param  {module:openmct.DomainObject} domainObject - Domain Object to use for telemetry collection
-     * @param  {object} options - Any options passed in for request/subscribe
+     * @param  {DomainObject} domainObject - Domain Object to use for telemetry collection
+     * @param  {Object} options - Any options passed in for request/subscribe
```

Comment:
We should define a standard `RequestOptions` type with the parameters defined in `standardizeRequestOptions()`. That way it's clear at a glance what options TC supports now